docs/user_guide/services/apic.rst

```diff
@@ -77,8 +77,11 @@ API to send POST command to the device.
       - Unique distinguished name describes the object and its place in the tree
       - Mandatory
     * - payload (dict)
-      - Dictionary containing the information to send via the post
+      - Information to send via the post command
```

Comment:
If `xml_payload=True`, then what will the `payload` argument be if XML data is passed in? Will it be a string?

Please update `* - payload (dict)` (line 79) if the data can be different data types.

Reply:
Done